src/compressed_tensors/compressors/sparse_compressors/sparse_24_bitmask.py

```diff
@@ -103,8 +103,10 @@ def from_compressed_data(
         :param bitmask: 2d bitmask of non-zero values
         :return: instantiated Sparse24BitMaskTensor
         """
-        if isinstance(shape, Tensor):
-            shape = shape.tolist()
+        if isinstance(shape, list):
```

Comment:
This is storing the original shape? Why would it be anything but a shape?

Reply:
Shape is a tensor of shape [2, 1]. Reasons:

1) we add a dimension for loading in vllm
2) we don't support loading in a `torch.Size` object directly (hence the tensor conversion)

TLDR; we convert torch.size object into torch.tensor and add a dimension

On CT side during decompression, we need to account for this conversion